sympy/ntheory/factor_.py

```diff
@@ -942,6 +942,15 @@ def factorint(n, limit=None, use_trial=True, use_rho=True, use_pm1=True,
     smoothness, smoothness_p, divisors
 
     """
+    if multiple:
+        fac = factorint(n, limit=limit, use_trial=use_trial,
+                           use_rho=use_rho, use_pm1=use_pm1,
+                           verbose=verbose, visual=False, multiple=False)
+        factorlist = sum(([p] * fac[p]
+                         if fac[p] > 0 else [S(1)/p]*(-1 * fac[p])
+                         for p in sorted(fac)), [])
```

Comment:
instead of straight sorting, perhaps `for p, _ in sorted(fac.items(), key=lambda (p, m): p if m > 0 else 1/p)` would be better so you would get `[1/7, 1/2, 5, 5]` for `factorrat(S(25)/14, multiple=True)` instead of `[1/2, 5, 5, 1/7]`.